crates/pyndakaas/docs/Makefile

```diff
@@ -0,0 +1,28 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+# TODO Added -E -a to force reload for autodoc, otherwise the re-generation of the build is very inconsistent!
```

Comment:
What is the downside of this? Is this something that should be resolved before merging?

Reply:
The downside of the flags is they clear the caches, so it has some overhead. I think it may be nice to get this command exactly right, so I wouldn't mind leaving this in as a low-priority todo.